core/templates/dev/head/player/flag_exploration_modal.html

```diff
@@ -0,0 +1,45 @@
+<script type="text/ng-template" id="modals/flagExploration">
+  <div class="modal-header">
+    <h3>
+      <span>Report this exploration</span>
```

Comment:
Hi @giritheja -- the way you internationalized the tooltip looks good. Could you do the same for all of these strings, using keys that start with I18N_PLAYER_REPORT_MODAL_*"? Then I will forward this to some of our other contributors to review the translations and suggest any necessary edits, but it makes more sense for them to review all the translations in one go.

Thanks!